bin/lint-migrations-file/src/lint_migrations_file.cljc

```diff
@@ -178,10 +177,8 @@
   :ok)
 
 (defn- migration-files []
-  #_:clj-kondo/ignore
-  (let [dir-str #?(:bb "resources/migrations" :clj "../../resources/migrations")
-        dir (io/file dir-str)]
-    (->> (file-seq dir)
+  (let [dir #_:clj-kondo/ignore (io/file #?(:bb "resources/migrations" :clj "../../resources/migrations"))]
+    (->> #_:clj-kondo/ignore (file-seq dir)
```

Comment:
Personally I would prefer keeping the linter on this whole let block, and just noting the particular linter rules as per [this suggestion](https://github.com/metabase/metabase/pull/58650#discussion_r2124945480).

Reply:
Look better now, @crisptrutski ?